Microsoft.DotNet.Interactive.PowerShell/PowerShellKernel.cs

```diff
@@ -118,6 +119,13 @@ public override bool TryGetVariable(string name, out object value)
             return false;
         }
 
+        public override void AddScriptReferences(IReadOnlyList<ResolvedPackageReference> resolvedReferences)
```

Comment:
@TylerLeonhardt Does this gesture make sense here? If not, we can move this method out of `KernelBase` and onto an interface for better composability. 

I'm inclined to do that anyway. This one feels like it won't apply to all possible kernels.

Reply:
So I'm curious what this gesture is about. Is it about adding dlls? If that's the case, then it might make sense.

cc @daxian-dbw

Reply:
It's about adding references to scripts. It makes sense for `.fsx` and `.csx` files. Would it make sense for `.ps1` files?

Reply:
> It's about adding references to scripts

What are these references? NuGet references?

Reply:
@TylerLeonhardt, 
no these are about adding **dll** references.
- in csi the equivalent is #r "c:\mydll.dll"
- in fsi it would #r @"c:\mydll.dll"

Kevin

Reply:
It's not that useful to PowerShell because we have the `Add-Type` cmdlet that takes in a path:

```pwsh
Add-Type -Path c:\mydll.dll
```